### ISSUE TITLE
Kuruma de DS (UZCJ) can't read any save data.

### ISSUE THREAD

**Issue by zurgeg** [melonDS-emu/melonDS#1683]
Upon loading, the game throws error 3 (データを読めませんでした。 電源を切って、DSカードを差し込 み直してください。)
This text translates to: "Data could not be read. Turn off the power, insert (?) the DS card"
Looking at it in No$GBA, it seems to be using some strange save form, as seen by the [card commands](https://gist.github.com/zurgeg/6105d3174cfc290b4f0706025637f4c9).

Looking at it, it's just reading random parts of the cartridge over and over again? I'll need to try on real HW at some point, but atm i don't have a JP DS so I can't test.

**Comment by RSDuck** [comment]
According to https://niwanetwork.org/wiki/Kuruma_de_DS it seems to have a bluetooth cartridge, maybe the same as Pokemon Typing Adventures.

> i don't have a JP DS so I can't test.

DS and DS lites don't have any region locking.

**Comment by zurgeg** [comment]
Ah alright, will try in a min. The error is internally labeled "SAVEDATA_ERROR_CARD_CONNECT" which is odd, if this error is about BT carts.

**Comment by RSDuck** [comment]
not necessarily, if the BT cart is like the IR carts, savedata is accessed a bit differently on them. So if the game tries to access saves before BT, that could already lead to a fail.

**Comment by zurgeg** [comment]
> not necessarily, if the BT cart is like the IR carts, savedata is accessed a bit differently on them. So if the game tries to access saves before BT, that could already lead to a fail.

Ahh, that would make a bit of sense. Do you know if Pokemon Typing does this too?

**Comment by zurgeg** [comment]
After looking at GH issues, looks like #225 may be related.

**Comment by zurgeg** [comment]
> Upon loading, the game throws error 3 (データを読めませんでした。 電源を切って、DSカードを差し込 み直してください。) This text translates to: "Data could not be read. Turn off the power, insert (?) the DS card" Looking at it in No$GBA, it seems to be using some strange save form, as seen by the [card commands](https://gist.github.com/zurgeg/6105d3174cfc290b4f0706025637f4c9).
> 
> Looking at it, it's just reading random parts of the cartridge over and over again? I'll need to try on real HW at some point, but atm i don't have a JP DS so I can't test.

Btw, just got everything set up on real HW, it has nothing to do with save data, as you expected.
